### DISCUSSION TITLE
Félicitations

### DISCUSSION THREAD

**Discussion by sebfar9172** [MadeInPierre/finalynx#109] [General]
Alors je voulais saluer le travail initié par @MadeInPierre car franchement plus je creuse plus je me dis que c'est un complément indispensable à Finary. Cela me permet d'avoir une visibilité sur ma stratégie investissement. 
J'utilise surtout la partie web qui est très claire.

**Comment by MadeInPierre** [comment]
Ça me fait très plaisir de recevoir ce message, merci beaucoup ! 😊

Et on est d'accord !! Le but des finances personnelles est de suivre une stratégie puis l'appliquer à des investissements placés dans la meilleure enveloppe pour chaque... Il faut suivre la stratégie, pas les enveloppes. Le tout pour atteindre nos projets de vie...

Tout ceci n'est donc que le début 😉 j'ai plein d'idées, le but final étant de décrire (1) nos objectifs de vie, (2) notre stratégie globale pour décrire notre aversion au risque et préférences d'investissement, et (3) nos rentrées d'argent comme les salaires. Avec ça, Finalynx devrait se débrouiller pour simuler l'évolution de notre patrimoine et sortir chaque mois combien investir dans quoi 🤞

Je t'invite à partager/mentionner ce projet quand tu peux 😇 je t'avoue que vu que c'est du 100% gratuit, j'aimerais bien que mon temps libre (et celui de tous les contributeurs dont toi !) serve au plus de gens possible ^^

**Comment by nmathey** [comment]
Oui bon boulot et en plus tu es patient avec moi :p 
Finalynx répond à pas mal de besoins soumis en idées à Finary mais dont certaines risques de mettre du temps à arriver, si elles arrivent...
je viens de faire un poste sur l'idée autour de la volonté d'afficher par poches personnalisée, mais je pense que le serait de faire une annonce sur le fil Finalynx du forum Finary quand il y a des "fonctionnalités majeures" ou répondant à des idées.

**Reply by MadeInPierre** [reply]
A part peut-être l'habitude de coder en Python, j'étais aussi débutant que toi sur tout le reste il y a 3 mois :p heureux de partager

Merci aussi ! En effet quand j'aurai le temps de me remettre sur des "vraies nouveautés" on partagera ça sur le post ^^
